app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,15 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="row.canDeploy()"
+                ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href=""
+                ng-if="!(row.canDeploy())"
+                ng-attr-aria-disabled="{{row.canDeploy() ? undefined : 'true'}}"
+                ng-class="{ 'disabled-link': !row.canDeploy() }">Deploy (paused)</a>
```

Comment:
No reason for the conditional checks here since you already have the `ng-if`. We can't say "(paused)" unless we check that's the reason it's disabled. It could also be being deleted for instance.

```html
<a
  href=""
  ng-if="!(row.canDeploy())"
  aria-disabled="true"
  class="disabled-link">Deploy</a>
```